.github/workflows/ci.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: "${{ matrix.ruby-version }}"
-        ruby-gems: "${{ matrix.ruby-version == '2.7' && 'latest' || default' }}"
+        rubygems: "${{ matrix.ruby-version == '2.7' && 'latest' || 'default' }}"
```

Comment:
alt:

```suggestion
        rubygems: latest
```

Reply:
cool. that is why I put it in there
Also, for some reason, would prefer to avoid the rubygem update if possible.